extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js

```diff
@@ -97,7 +97,7 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
     }
 
     $scope.getMinText = function(item) {
-        if (!item.stateDescription || !item.stateDescription.minimum) {
+        if (!item.stateDescription || isNaN(item.stateDescription.minimum)) {
```

Comment:
Yes, explict null and empty string check "" or this: isNaN(parseInt(item.stateDescription.minimum))